src/makeCalculator.js

```diff
@@ -5,6 +5,43 @@
  */
 function makeCalculator() {
   // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+    add(num) {
+      result += num;
+
+      return this;
+    },
+    subtract(num) {
+      result -= num;
+
+      return this;
+    },
+    multiply(num) {
+      result *= num;
+
+      return this;
+    },
+    divide(num) {
+      result /= num;
```

Comment:
The `divide` method does not handle division by zero. Consider adding a check to prevent division by zero, which could result in `Infinity` or `NaN`.